contracts/remappings.txt

```diff
@@ -1,2 +1,3 @@
 @subnet-evm-contracts=lib/subnet-evm/contracts/contracts
-@openzeppelin/=lib/openzeppelin-contracts/
\ No newline at end of file
+@openzeppelin/=lib/openzeppelin-contracts/
+@teleporter/=src/Teleporter/
```

Comment:
Do we also have to add this to `slither.config`?

Reply:
turns out slither with a foundry project already detects the remappings so no need to. Also removed extra solc remaps we previously had in `slither.confg`